lib/utils/note-utils.js

```diff
@@ -7,7 +7,7 @@
  *
  * @type {RegExp} matches a title and excerpt in note content
  */
-const noteTitleAndPreviewRegExp = /(\S[^\n]*)\s*(\S[^\n]*)?/;
+const noteTitleAndPreviewRegExp = /()(\S[^\n]*)\s*(\S[^\n]*)?/;
```

Comment:
is this change a typo here?

Reply:
No. This is intended. Standard library for RegEx doesn't have named groups so I had to add a fake empty group at the beginning so in further part we can refer to the same indexes for notes in markdown and not. Few lines below I've changed indexes to match new regex [1] -> [2], [2] -> [3]